namespaces/live.cloud-platform.service.justice.gov.uk/data-platform-app-ltc-capabilites-app-dev/00-namespace.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
     cloud-platform.justice.gov.uk/is-production: "false"
     cloud-platform.justice.gov.uk/environment-name: "dev"
   annotations:
-    cloud-platform.justice.gov.uk/business-unit: "Platforms"
-    cloud-platform.justice.gov.uk/slack-channel: "data-platform-application-migration-support"
+    cloud-platform.justice.gov.uk/business-unit: "HQ"
+    cloud-platform.justice.gov.uk/slack-channel: ""
     cloud-platform.justice.gov.uk/application: "ltc-capabilites-app"
-    cloud-platform.justice.gov.uk/owner: "analytical-platform: analytics-platform-tech@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/owner: "People Analytics"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/ltc-capabilites-app"
-    cloud-platform.justice.gov.uk/team-name: "analytics-hq"
+    cloud-platform.justice.gov.uk/team-name: ""
```

Comment:
This is missing from the spreadsheet - should we wait for it to be completed, or I can add `people-analytics`?